### PR TITLE
setupaliases.lic: add lfr alias to look at raw data of remote rooms.

### PR DESCRIPTION
**Pull request by Sarvatt**
usage: lfr <roomnumber>. 

Happy to change the name, in my head its look far room instead of just look room like the lr alias.

### CODE REVIEW

File: setupaliases.lic [hunk 1]
```diff
@@ -22,6 +22,9 @@
   # echos the full data of the room you're standing in
   ['lr', "#{$clean_lich_char}e echo Room[Room.current.id].inspect"],
 
+  # echos the full data of a specified remote room
+  ['lfr', "#{$clean_lich_char}e echo Room[\\?].inspect"],
```
Comment (rpherbig) [inline setupaliases.lic]: I expected this to be `\?` not `\\?` - did you run this to be sure?

**Comment by jandersson** [comment]
Is there any way to have `lr` handle an optional argument? As in typing `lr` looks at the current room and `lr 1234` looks at room 1234. Just a thought, I don't know if its possible.

File: setupaliases.lic (hunk 1 above)
Comment (jandersson) [reply setupaliases.lic]: I believe its to escape the first slash so ;alias can insert the argument there. See the `fz` command

Comment (jandersson) [reply setupaliases.lic]: `fz` was broken until sarvatt added the second slash

**Comment by jandersson** [comment]
Not trying to hold this up by the above question - just brainstorming `lfr 2323` or `lr 2323` works great for me.

File: setupaliases.lic (hunk 1 above)
Comment (Sarvatt) [reply setupaliases.lic]: yeah I ran it first to be sure, had to fix up a few others not working without the extra escape

Comment (jandersson) [reply setupaliases.lic]: I added the `fz` command manually via ;alias then assumed it would work with ;setupaliases without any changes. Good fix 👍

**Pull request merged**